### DISCUSSION TITLE
Kodi NFO Metadata Missing Trailer Field and others

### DISCUSSION THREAD

**Discussion by neo-neo1** [pymedusa/Medusa#10098] [Ideas & Feature requests]
Kodi (Kodi 12+) NFO metadata lacks a few fields. Comparing Kodi metadata NFO files generated by Sonarr (and Jellyfin/Emby) this becomes evident. 

[Several fields missing](https://kodi.wiki/view/NFO_files/TV_shows) such as `trailer`, `status`, and etc...

Omiting `trailer` field disables ability for Kodi (and others) to provide clip previews. If indexer provides it please include it.

Kodi Trailer Field Syntax: `<trailer>plugin://plugin.video.youtube/play/?video_id=IDHERE</trailer>` The rest are straight forward.

Thank You

**Comment by p0psicles** [comment]
We can only place data in the nfo that we also keep stored in the db for the show.
We don't store the trailer url. We do have info on the status. So I could take a look at that if we don't send it already.

**Comment by netty0** [comment]
> We can only place data in the nfo that we also keep stored in the db for the show.
>
> We don't store the trailer url. 

It’s unfeasible to scrape trailers? Disappointing.


> We do have info on the status. So I could take a look at that if we don't send it already.
>

Please also cross reference other missing fields with the link.

A family member noticed TV show information is less detailed when compared to (Radarr generated) movie information. They make use of trailers in Kodi often.

**Comment by p0psicles** [comment]
yeah well it's not impossible to send additional data. Just have to go back to the indexer's api to get more data. But this is not an issue, but more a feature request.

**Reply by papampi** [reply]
So ... is getting  more fields from db into nfo files still in plan?

**Comment by papampi** [comment]
Kodi TV Show info generated by Medusa and TinyMediaManager:


![Screenshot from 2023-08-29 16-52-24](https://github.com/pymedusa/Medusa/assets/1108483/d3bff9fb-5e8c-47b0-a3fc-bae65bb50906)
![Screenshot from 2023-08-29 16-55-22](https://github.com/pymedusa/Medusa/assets/1108483/f4b69f4b-fa1e-4592-ba9e-090b315ccc93)
